include/xgboost/data.h

```diff
@@ -487,6 +498,11 @@ class DMatrix {
                          const std::string& cache_prefix = "",
                          const size_t page_size = kPageSize);
 
+  static DMatrix* CreateMultiple(std::vector<std::shared_ptr<dmlc::Parser<uint32_t>>> parsers,
```

Comment:
Should we overload the `Create` function instead of using a new name `CreateMultiple`?

Reply:
Done